react/features/app-settings/components/AppSettings.native.js

```diff
@@ -23,55 +28,76 @@ import { translate } from '../../base/i18n';
  */
 class AppSettings extends AbstractAppSettings {
 
+    /**
+    * Instantiates a new {@code AppSettings} instance.
+    *
+    * @inheritdoc
+    */
+    constructor(props) {
+        super(props);
+
+        this._getSafetyPadding = this._getSafetyPadding.bind(this);
+    }
+
     /**
      * Implements React's {@link Component#render()}, renders the settings page.
      *
      * @inheritdoc
      * @returns {ReactElement}
      */
     render() {
-        const { t } = this.props;
+        const { _profile, t } = this.props;
```

Comment:
deconstructing _profile may also be worth it here

Reply:
I'm not sure it is. Reading the code, "profile.defaultURL" is something I understand immediately, while a variable simply named "defaultURL" may have multiple meaning.